src/api/init.ts

```diff
@@ -79,6 +83,26 @@ export async function startApiServer(datastore: DataStore): Promise<ApiServer> {
   // Setup direct proxy to core-node RPC endpoints (/v2)
   app.use('/v2', createCoreNodeRpcProxyRouter());
 
+  // Rosetta API -- https://www.rosetta-api.org
+  app.use(
+    '/rosetta/v1',
+    (() => {
+      const router = addAsync(express.Router());
+      router.use(cors());
+      router.use('/network', createRNetworkRouter(datastore));
```

Comment:
nit: I prefer more explicit naming e.g. `createRNetworkRouter` vs `createRosettaNetworkRouter`. Thoughts?

Reply:
The [original commit](https://github.com/fdmirza/stacks-blockchain-rosetta-api/commit/47db1a61cf6bd014a42c39952af3e9e3f3651f34#diff-f0977764d44e25489d36496dcb219a87) for this had the longer name, but was not consistent.  I shortened it for this PR; I can change them all to say `Rosetta` instead of `R` everywhere.